src/actions/validate.js

```diff
@@ -26,28 +27,32 @@ const validateData = (schema, data) => {
   }
 };
 
-const generateSchema = (requiredLanguages, optionalLanguages) => {
+const generateSchema = (requiredLanguages, optionalLanguages, collections) => {
   let schema = schemaTemplate;
+  
+  const upperCaseCollections = collections.map(s => s.toUpperCase());
```

Comment:
There's a bunch of places where we convert lowercase outputTypes/collections to uppercase (or the other way around). Also, the public APIs are not really consistent. For example:

Rendering using the CLI command requires lowercase outputTypes:
`localicious render ...-outputTypes android`

While in the LocaliciousConfig.yaml we use uppercase:
```
...
outputTypes:
  - IOS
  - ANDROID
```

Should we maybe enforce uppercase everywhere?